kube/services/jobs/usersync-cronjob.yaml

```diff
@@ -73,6 +81,27 @@ spec:
                   configMapKeyRef:
                     name: manifest-global
                     key: sync_from_dbgap
+                    optional: true
+              - name: DBGAP_BACKUP_BUCKET
+                valueFrom:
+                  configMapKeyRef:
+                    name: manifest-global
+                    key: dbgap_backup_bucket
+                    optional: true
+              - name: DBGAP_DOWNLOAD_FOLDER
+                valueFrom:
+                  configMapKeyRef:
+                    name: manifest-global
+                    key: dbgap_download_folder
+                    optional: true
```

Comment:
Why are we making this a variable rather than just hard coding a fixed path?

Reply:
+1